### ISSUE TITLE
Why do you have normalization layer in between separable conv?

### ISSUE THREAD

**Issue by s1ddok** [mit-han-lab/gan-compression#19]
I spotted that you have a normalization layer in your separable convolution implementation. 

```python
self.conv = nn.Sequential(
       nn.Conv2d(in_channels=in_channels, out_channels=in_channels * scale_factor, kernel_size=kernel_size,
                      stride=stride, padding=padding, groups=in_channels, bias=use_bias),
       norm_layer(in_channels),
       nn.Conv2d(in_channels=in_channels * scale_factor, out_channels=out_channels,
                      kernel_size=1, stride=1, bias=use_bias),
)
```

I did not see such implementations before. Also, why it doesn't get adjust to `scale_factor`?

**Comment by tonylins** [comment]
Hi, adding a normalization layer inside the separable convolution is also quite common practice (e.g., see https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenet.py#L58), and it should not affect the performance too much.

The `norm_layer(in_channels)` seems to be a bug here, it should be `norm_layer(in_channels * scale_factor)`. Since we always use `scale_factor=1`, the bug is not exposed. Thanks for pointing out and we will fix it soon.
